src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -700,16 +700,16 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         this->Focus(FocusState::Programmatic);
     }
 
-    winrt::fire_and_forget TermControl::UpdateControlSettings(IControlSettings settings)
+    void TermControl::UpdateControlSettings(IControlSettings settings)
     {
-        return UpdateControlSettings(settings, _core.UnfocusedAppearance());
+        UpdateControlSettings(settings, _core.UnfocusedAppearance());

```

Comment:
📝 This is really the only change in this PR that isn't `winrt::fire_and_forget` --> `safe_void_coroutine`. Should be fine though.